soup_test.py

```diff
@@ -0,0 +1,33 @@
+from bs4 import BeautifulSoup
+from urllib.request import urlopen
+
+url = "https://lpi.oregonstate.edu/mic/glossary" #specify source location
+page = urlopen(url)
```

Comment:
There is a chance that there could be network errors that would cause this part to fail. We should add some exception handlers such as "HTTPError" and "URLError" to occur if the MIC server is down or the URL is incorrect or fails to reach its destination. This will make our debugging and that of the later teams much easier. This is a very simple app, so it wouldn't really be that hard to find the problem, but we should add some error detection early on just to follow good practices.